xbmc/settings/SettingControl.cpp

```diff
@@ -226,6 +226,8 @@ bool CSettingControlList::Deserialize(const TiXmlNode *node, bool update /* = fa
   XMLUtils::GetInt(node, SETTING_XML_ELM_CONTROL_HEADING, m_heading);
   XMLUtils::GetBoolean(node, SETTING_XML_ELM_CONTROL_MULTISELECT, m_multiselect);
   XMLUtils::GetBoolean(node, SETTING_XML_ELM_CONTROL_HIDEVALUE, m_hideValue);
+  XMLUtils::GetBoolean(node, SETTING_XML_ELM_CONTROL_ADDVALUE, m_addlistvalue);
+  XMLUtils::GetInt(node, SETTING_XML_ELM_CONTROL_ADDBUTTONLABEL, m_addButonLabel);
```

Comment:
Buton with one t?
might be deliberate, but just thought i'd mention it.

Reply:
Thanks, a typo that I cut'n'pasted everywhere. Bah!